### ISSUE TITLE
Lookup Symbol isn't accurate on the first use (regression in 4.0 alpha 4)

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#59820]
### Godot version

7bb963e

### System information

Windows 10 x64

### Issue description

![godot windows tools 64_PsqVCHivTA](https://user-images.githubusercontent.com/2223172/161406124-d3ea3d2a-8c9d-4831-a40b-4ea7ed7bfec1.gif)
Notice how on the first jump the method was nowhere in focus. It worked on the second click, when the doc was already opened.

### Steps to reproduce

1. Add code with some built-in class' method
2. Ctrl + Click that method
3. You are thrown somewhere into editor help, but the method is not in sight

**Comment by miv391** [comment]
This bug is still present in v4.0.alpha7.official [3e9ead05f]

**Comment by Calinou** [comment]
> This bug is still present in v4.0.alpha7.official [[3e9ead0](https://github.com/godotengine/godot/commit/3e9ead05f2e87e46b5982cc9a140e172ee98c227)]

Can you post the code snippet you used to trigger the issue?

**Comment by miv391** [comment]
I can trigger the issue using this code by Ctrl-clicking any of the functions:
```gdscript
func test():
	'foo'.begins_with('f')
	'foo'.ends_with('o')
	[1,2].sort()
```
The bug happens only once per document page (in this case String and Array pages). So the bug happens once for either `begins_with` or `ends_with` and once for `sort`. Any subsequent Ctrl-clicks to already opened page work and the correct location on the page is shown. Closing the page after the first use does not affect the situation, subsequent Ctrl-click still works.

At first I was going to report a new issue:

> Search Help shows wrong location on first search
> 
> - start Godot
> - press F1 to open Serch Help window
> - search "sort" (Array/sort)
> 
> Array's document page opens, but the page location is wrong. Instead of displaying the sort method, bottom of the page is shown. This happens always with the first search. Subsequent searches show the correct location as expected.

But my guess is that these bugs have a same root cause.

**Comment by KoBeWi** [comment]
I tried the above code and can't reproduce the issue. The only problem is that the view isn't centered on the function.

**Comment by miv391** [comment]
This is what I see when I Ctrl-click `begins_with` first time. Page is scrolled to the bottom. Second Ctrl-click shows the correct page position. (I'm using Win64 version.)

![kuva](https://user-images.githubusercontent.com/49998025/166103868-897778ea-61ee-4922-8c84-6da4d3428233.png)

**Comment by KoBeWi** [comment]
Maybe this depends on the resolution/UI scale.

**Comment by miv391** [comment]
My monitor has 3440 x 1440 resolution, Windows 10 is using 100% scaling and also Godot is using the default 100% Display Scale. Chaging the Display Scale doesn't change how this works. The window size of Godot doesn't change anything either.

Godot v3.4.4.stable.official [419e713a2] works as expected and always shows the correct page location.

**Comment by miv391** [comment]
I tested different alpha versions:

v4.0.alpha3.official [256069eaf] works (also alpha1 and alpha2 work)
v4.0.alpha4.official [f47097973] bugs

**Comment by miv391** [comment]
This works on `v4.0.beta4.official [e6751549c]`. I don't know which exact version fixed this, but at least with this version you jump to the correct place on the first click.
